config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -392,6 +395,9 @@ spec:
                 postgres_storage_class:
                   description: Storage class to use for the PostgreSQL PVC
                   type: string
+                postgres_priority_class:
+                  description: Assign a preexisting priority class to the postgres pod
+                  type: string
```

Comment:
This should be an integer - https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass